chapter_13_database_layer_validation.asciidoc

```diff
@@ -876,7 +852,6 @@ Ran 4 tests in 9.951s
 OK
 ----
 
-// DAVID: I get four "Not Found: /favicon.ico" messages when running the FTs, just FYI.
```

Comment:
me too, but I think it's no big deal. I think the HTML is referencing it by default.